core-bundle/tests/EventListener/SearchIndexListenerTest.php

```diff
@@ -99,47 +99,81 @@ public function getRequestResponse(): \Generator
             false,
         ];
 
-        yield 'Should be deleted because the response was not successful (404)' => [
+        yield 'Should be deleted because the response was "not found" (404)' => [
             Request::create('/foobar'),
             new Response('', 404),
             SearchIndexListener::FEATURE_DELETE | SearchIndexListener::FEATURE_INDEX,
             false,
             true,
         ];
 
-        yield 'Should be deleted because the response was not successful (403)' => [
+        yield 'Should be deleted because the response was "gone" (410)' => [
             Request::create('/foobar'),
-            new Response('', 403),
+            new Response('', 404),
             SearchIndexListener::FEATURE_DELETE | SearchIndexListener::FEATURE_INDEX,
             false,
             true,
         ];
 
-        yield 'Should not be deleted because even though the response was not successful (403), it was disabled by the feature flag ' => [
+        yield 'Should not be deleted because even though the response was "not found" (404), it was disabled by the feature flag ' => [
             Request::create('/foobar'),
-            new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"Page","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>', 403),
+            new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"Page","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>', 404),
             SearchIndexListener::FEATURE_INDEX,
             false,
             false,
         ];
 
-        $response = new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"Page","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>', 403);
+        $response = new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"Page","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>', 200);
         $response->headers->set('X-Robots-Tag', 'noindex');
 
-        yield 'Should not be handled because the X-Robots-Tag header contains "noindex" ' => [
+        yield 'Should not index but should delete because the X-Robots-Tag header contains "noindex" ' => [
```

Comment:
```suggestion
        yield 'Should not index but should delete because the X-Robots-Tag header contains "noindex"' => [
```

Reply:
They look like a mistake to me. But I can test this myself if you want.

Reply:
I think they were only a typo. I'll remove them everywhere.

Reply:
Thank you. 🙏